Cartfile.resolved

```diff
@@ -1,2 +1,3 @@
-github "ReactiveX/RxSwift" "3.0.1"
-github "daltoniam/Starscream" "2.0.2"
+github "ReactiveX/RxSwift" "3.3.1"
+github "RxSwiftCommunity/RxSwiftExt" "2.1.0"
```

Comment:
I have no idea how it happened. In `Cartfile` there is no entry with `RxSwiftExt`. I'll fix this